internal/controllers/server/tests/import-server/01-assert.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Server
+metadata:
+  name: import
+status:
+  conditions:
+    - type: Available
+      message: OpenStack resource is available
```

Comment:
There's [your error](https://github.com/k-orc/openstack-resource-controller/actions/runs/13364878654/job/37320577976?pr=254#step:7:555):
```
    case.go:398: resource Server:kuttl-test-pure-hawk/import: .status.conditions.message: value mismatch, expected: OpenStack resource is available != actual: OpenStack resource is up to date
```

The server is still in BUILD status after the allocated time, and has thus not moved to available == true. You should bump the timeout for this step.

Reply:
How can I bump up the timeout just in this step? I think it's only valid in TestAssert